resources/qml/Avatar.qml

```diff
@@ -53,12 +53,12 @@ Rectangle {
 		height: avatar.height / 6
 		width: height
 		radius: settings.avatarCircles ? height / 2 : height / 4
-		color: switch (timelineManager.userPresence(userid)) {
-			case "online": return "#00cc66"
-			case "unavailable": return "#ff9933"
-			case "offline": return "#a82353"
-			default: "transparent"
-		}
+		// color: switch (timelineManager.userPresence(userid)) {
```

Comment:
Will want to un-comment this when you're finished

Reply:
userPresence was weirdly causing ```Aborted (core dumped)```, Sure I'll fix it before it is finished

Reply:
I fixed this for you, but you would need to pull this commit: https://github.com/Nheko-Reborn/nheko/commit/8f6281a6b05a47fd46a1bed51a1d88e8ec74943d (I couldn't push it to your branch)